numba_cuda/numba/cuda/tests/cudapy/test_bfloat16.py

```diff
@@ -0,0 +1,50 @@
+from numba import cuda, float32
+from numba.cuda.bf16 import bfloat16
+from numba.cuda.testing import unittest, CUDATestCase
+
+import math
+
+
+@unittest.skipIf(
+    not cuda.is_bfloat16_supported(),
```

Comment:
(similar note to the one on the `skipIf()` in `test_bfloat16_bindings.py` below)

Reply:
Done [144497e](https://github.com/NVIDIA/numba-cuda/pull/245/commits/144497e63447c055e1a39b5907c1aef6a790e31a)